extra/fr.inria.Coccinelle.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2024 INRIA, University of Copenhagen DIKU -->
+<component type="console-application">
+  <id>fr.inria.Coccinelle</id>
+  <name>Coccinelle</name>
+  <summary>Match and transform C code using semantic patches</summary>
+  <developer id="fr.inria.paris.who.lawall.julia">
+    <name>Julia Lawall</name>
+  </developer>
+  <metadata_license>CC-BY-SA-4.0</metadata_license>
+  <project_license>GPL-2.0-only</project_license>
+  <description>
+    <p>Coccinelle is a program matching and transformation engine which provides the language SmPL (Semantic Patch Language) for specifying desired matches and transformations in C code. Coccinelle was initially targeted towards performing collateral evolutions in Linux. Such evolutions comprise the changes that are needed in client code in response to evolutions in library APIs, and may include modifications such as renaming a function, adding a function argument whose value is somehow context-dependent, and reorganizing a data structure. Beyond collateral evolutions, Coccinelle is successfully used (by us and others) for finding and fixing bugs in systems code.</p>
```

Comment:
Note: We should check who the copyright holder is or replace it with a new text.

Reply:
We decided to go with CC on the ML: https://lore.kernel.org/cocci/6f6d90f8-1027-46ed-a518-0c3b4d6285d6@inria.fr/